Conference-Android/app/src/main/java/com/sagetech/conference_android/app/ui/activities/ConferenceSessionListActivity.java

```diff
@@ -73,7 +73,9 @@ public boolean onOptionsItemSelected(MenuItem item) {
         switch (item.getItemId()) {
             // Respond to the action bar's Up/Home button
             case android.R.id.home:
-                NavUtils.navigateUpFromSameTask(this);
+                Intent intent = NavUtils.getParentActivityIntent(this);
```

Comment:
this probably should change back if it is relating to what I spoke of earlier in the AndroidManifest.xml